vars/spark_vars.yml.template

```diff
@@ -2,9 +2,6 @@
 #Define sbt version
 sbt_version: 0.13.15
 
-#Define Scala release
-scala_release: 2.11
```

Comment:
I still see the var being used in https://github.com/nlesc-sherlock/emma/blob/remote_debugging/roles/spark/tasks/main.yml#L99

So should it be removed?